.eslintrc

```diff
@@ -128,6 +128,7 @@
     "import/prefer-default-export": 0,
 
     /* || jsx-a11y plugin || */
+    "jsx-a11y/label-has-for": 0,
```

Comment:
:grimacing: Since this deprecated rule has a [replacement](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md), I'd like to see if we can switch to it and fix any of the new errors.